### PR TITLE
Chat event hook guide

### PR DESCRIPTION
**Pull request by howawong**
Add Chat cloud function event hook guides in Python and JS

Connects #166

### CODE REVIEW

**Comment by carmenlau** [comment]
Content looks good to me. Pending for verifying the code work in skygear code.
1. I think there will be error in `npm install` that we need adding `skygear` in package.json
2. Please also remember to update the demo repo if code update is needed.

Thanks @howawong 🙏🏻

File: chat/cloud-event-hook/python.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+from chat.hooks import *
+
+@after_message_sent
+def after_message_sent_hook(message, conversation, participants):
+    container = SkygearContainer(api_key=skyoptions.masterkey,
+                                 user_id=current_user_id())
+    message = deserialize_record(message)
+    conversation = deserialize_record(conversation)
+    participants = [deserialize_record(p) for p in participants]
+    other_user_ids = [p.id.key for p in participants if p != current_user_id()]
```
Comment (rickmak) [inline chat/cloud-event-hook/python.md]: Please avoid one-liner

**Comment by howawong** [comment]
@carmenlau  yes, correct 🙈

**Comment by carmenlau** [comment]
@howawong Would like to know the status of js example 🙈

**Comment by howawong** [comment]
Both Guide PR and cloud code JS PR are updated.

**Comment by carmenlau** [comment]
Per discussion, we will use the message user id instead of the context current user in the demo. Please update @howawong, thanks!

**Comment by carmenlau** [comment]
Please add update skygear-doc for the sidebar menu

**Review by carmenlau — APPROVED** [review]

**Pull request merged**